packages/core/utils/src/inventory/events.ts

```diff
@@ -1,14 +1,13 @@
-import { CommonEvents } from "../event-bus"
+import { buildEventNamesFromEntityName } from "../event-bus"
+import { Modules } from "../modules-sdk"
 
-export const InventoryEvents = {
-  created: "inventory-item." + CommonEvents.CREATED,
-  updated: "inventory-item." + CommonEvents.UPDATED,
-  deleted: "inventory-item." + CommonEvents.DELETED,
-  restored: "inventory-item." + CommonEvents.RESTORED,
-  reservation_item_created: "reservation-item." + CommonEvents.CREATED,
-  reservation_item_updated: "reservation-item." + CommonEvents.UPDATED,
-  reservation_item_deleted: "reservation-item." + CommonEvents.DELETED,
-  inventory_level_deleted: "inventory-level." + CommonEvents.DELETED,
-  inventory_level_created: "inventory-level." + CommonEvents.CREATED,
-  inventory_level_updated: "inventory-level." + CommonEvents.UPDATED,
-}
+const eventBaseNames: ["inventoryItem", "reservationItem", "inventoryLevel"] = [
+  "inventoryItem",
+  "reservationItem",
+  "inventoryLevel",
+]
```

Comment:
Wouldn't `as const` do the same without having to explicitly list the eventnames?